requirements-dev.txt

```diff
@@ -1,2 +1,6 @@
+nose>=1.3.7
+nose-parameterized>=0.5.0
 flake8
 flake8-import-order
+sphinx>=1.4
+sphinx_rtd_theme>=0.2.2
```

Comment:
would this require user to install the dev version of pyradiomics before documentation can be built? (or have users install `sphinx` and `sphinx-rtd-theme` manually?)